client/CMakeLists.txt

```diff
@@ -168,6 +169,7 @@ set(client_HEADERS
 		mapHandler.h
 		resource.h
 		SDLRWwrapper.h
+        ClientCommandManager.h
```

Comment:
I assumed that you will fix spaces in both places, but seems like need to leave comment separately :)